src/realm/sync/client.cpp

```diff
@@ -1226,17 +1226,27 @@ void SessionWrapper::report_progress()
     ClientHistory::get_upload_download_bytes(m_db.get(), downloaded_bytes, downloadable_bytes, uploaded_bytes,
                                              uploadable_bytes, snapshot_version);
 
+    // In protocol versions 25 and earlier, downloadable_bytes was the total
```

Comment:
This comment is nonsensical wherever it is and I should have asked to remove it before. The current protocol version is 3. There is no protocol version 25 or protocol version 26. They are dead and buried deep in the ground.

We do this addition here to hide the real meaning of these values from object-store.